test/gpu/autoswitch.jl

```diff
@@ -0,0 +1,14 @@
+using OrdinaryDiffEq, CUDA, Test
```

Comment:
If you add a new test file then you need to add it to runtests.jl

Reply:
Ah right, thanks!